### PR TITLE
Og images

### PR DESCRIPTION
**Pull request by dominikwilkowski**

### CODE REVIEW

File: code/layout/head.js [hunk 1]
```diff
@@ -30,16 +30,16 @@ const Head = ({ _ID, pagetitle }) => (
...
 		<meta property="og:title" content="Design System Meetup"/>
-		<meta property="og:description" content="Join us for all things Design System. From solving issues in multi platform environments to how we name things this is where we share what we've learned building large and small scale design systems. We meet every second Monday of every second month."/>
+		<meta property="og:description" content={ eventdescription }/>
 		<meta property="og:url" content="https://designsystemmeetup.com/"/>
-		<meta property="og:image" content="https://designsystemmeetup.com/assets/img/og-image.jpg"/>
+		<meta property="og:image" content={`https://designsystemmeetup.com/assets/img/${ image }`}/>
 		<meta property="og:image:height" content="1257"/>
 		<meta property="og:image:width" content="2400"/>
 		<meta name="twitter:card" content="summary_large_image"/>
 		<meta name="twitter:title" content="Design System Meetup Sydney"/>
```
Comment (alex-e-leon) [inline code/layout/head.js:39]: FYI - Twitter will use most OG tags as well - so you don't need to define all the twitter meta properties as well:
https://css-tricks.com/essential-meta-tags-social-media/

Comment (dominikwilkowski) [reply code/layout/head.js:39]: not just for twitter or fb... also linkedin and twitter etc... this is supposedly the the best set: https://medium.com/slack-developer-blog/everything-you-ever-wanted-to-know-about-unfurling-but-were-afraid-to-ask-or-how-to-make-your-e64b4bb9254

Comment (alex-e-leon) [reply code/layout/head.js:39]: That site doesn't mention that twitter will use og tags though, so some of them aren't required to be defined twice.
So you could probably delete `twitter:image`, `twitter:description` `twitter:title` and `twitter:name` (`twitter:name` isn't even valid) - as the rest are all already defined in og tags.
and you also have `og:image:width` and `og:image:height` defined twice

Comment (dominikwilkowski) [reply code/layout/head.js:39]: yeah its not about twitter.. it's about the other services that use a combination of each... won't hurt to have them in there twice no?

**Comment by alex-e-leon** [comment]
Doesn't hurt but not needed either. They aren't doing anything

**Pull request merged**